adobe/spectrum-web-components
# docs: use latest Storybook Prebuilt and leverage the actions addon

## Description
Less `console.log()` and more built in features of Storybook makes testing simpler.

## Motivation and context
`console.log()` leaks into shared contexts where the CSF exports are leveraged, like tests.

## How has this been tested?

    1. Go to https://storybook--spectrum-web-components.netlify.app/storybook/index.html?path=/story/checkbox-sizes--s
    2. Toggle the checkbox
    3. See `change` and `click` reports in the "Actions" tab
    1. Go to https://storybook--spectrum-web-components.netlify.app/storybook/index.html?path=/story/color-area--default
    2. Drag the color handle around
    3. See `input` repots in the "Actions" tab
- Etc.

## Types of changes

## Checklist
